dvc/tree/base.py

```diff
@@ -59,6 +59,8 @@ class BaseTree:
     TRAVERSE_PREFIX_LEN = 3
     TRAVERSE_THRESHOLD_SIZE = 500000
     CAN_TRAVERSE = True
+
+    # Needed for some providers, and http open()
     CHUNK_SIZE = 64 * 1024 * 1024  # 64 MiB
```

Comment:
Looks like it is now only used in http, so I guess no need to have it in the base class.
```suggestion
```

Reply:
It's used in the `open()` call of `_transfer_file()`